python/src/nanoarrow/ipc.py

```diff
@@ -236,9 +243,198 @@ def __repr__(self) -> str:
             return f"<{class_label} <invalid>>"
 
 
+class Writer:
```

Comment:
Add a class docstring for Writer?

Reply:
Also, to understand a bit the different classes here. The already existing `Stream` object is essentially the "reader" equivalent?

And for this writer, does that only support the stream format or also the file format?